code/modules/events/grey_tide.dm

```diff
@@ -0,0 +1,61 @@
+GLOBAL_LIST_EMPTY_TYPED(grey_tide_areas, /area)
+
+/datum/round_event_control/grey_tide
+	name = "Grey Tide"
+	typepath = /datum/round_event/grey_tide
+	max_occurrences = 2
+	min_players = 5
+	category = EVENT_CATEGORY_ENGINEERING
+	description = "Bolts open all doors in one or more departments."
+
+/datum/round_event_control/grey_tide/can_spawn_event(players_amt)
+	. = ..()
+
+	for(var/datum/round_event/running_event in SSevents.running)
+		if(istype(running_event, /datum/round_event/grey_tide)) //Two of these at once messes up the list
+			return FALSE
+
+/datum/round_event/grey_tide
+	announce_when = 50
+	end_when = 20
+	var/severity = 1
+
+/datum/round_event/grey_tide/setup()
+	announce_when = rand(50, 60)
+	end_when = rand(20, 30)
+	severity = rand(1,3)
+
+	var/list/potential_areas = list(/area/station/command,
+		/area/station/engineering,
+		/area/station/medical,
+		/area/station/security,
+		/area/station/cargo,
+		/area/station/science,
+	)
+
+	for(var/i in 1 to severity)
+		GLOB.grey_tide_areas += pick_n_take(potential_areas)
```

Comment:
I don't think we should be putting this in a glob, we theoretically want to be able to support multiple at once.

Instead, create the list, and send that list through the signal.

Reply:
Need to do this still